pkg/querier/queryrange/split_by_test.go

```diff
@@ -134,7 +151,7 @@ func TestSplitQuery(t *testing.T) {
 		},
 	} {
 		t.Run(strconv.Itoa(i), func(t *testing.T) {
-			days := splitQuery(tc.input)
+			days := splitQuery(tc.input, 24*time.Hour)
```

Comment:
Would be nice to see a test with a different interval as well, otherwise there's no real guarantee it works with custom intervals.

Reply:
ho good point here.